Gemfile

```diff
@@ -57,4 +57,5 @@ gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
 
 gem 'carrierwave'
 
+gem 'i18n_generators'
```

Comment:
これ何かを参考にして追加したんでしょうか? 使うなとは言いませんが、ちゃんと理解するまではlocaleファイルは自力で書く練習したほうがいいんじゃないかなと思います。

それとこのgemのREADMEを読みましたか?

```
Restricting the group to :development would be a good idea, since the generator would probably be used only in development mode.
```
とあります。そのように対応しましょう。

参考
https://github.com/amatsuda/i18n_generators

Reply:
プラクティスのヒントに
> i18n_generators gemを使うと、翻訳ファイルの作成がちょっと楽になるかも？

と書いてあったので使っていました。でも確かに自分で書いた方が理解力が上がりそうですね。
自分で書くようにしてみます！

> Restricting the group to :development would be a good idea, since the generator would probably be used only in development mode.

すみません、この文章がよく理解できませんでした。
Railsの構造に合わせて入れ子構造で書くことが推奨されているということでしょうか？

Reply:
Google翻訳ですが、
> ジェネレーターはおそらく開発モードでのみ使用されるため、グループを：developmentに制限することをお勧めします。

といってます。つまり

Gemfileで以下のようにしましょうってことです。Railsにはデフォルトでproductionモード、testモード、developmentモードがあります。`group :development do`に書くとdevelopmentモードでだけ有効になります。i18n_generatorsはi18nファイルを作ることだけがお仕事です。Webアプリケーションとして機能させるために必要なものではないのでproduction、つまり本番環境では読み込む必要がないってことですね。
```ruby
group :development do
  gem 'listen', '~> 3.3'
  gem 'rack-mini-profiler', '~> 2.0'
  gem 'spring'
  gem 'web-console', '>= 4.1.0'
  <- ここにi18n_generatorsを追加
end
```
という、

Reply:
そういう意味だったんですね！
翻訳してもdevelopmentの意味がわからず対応できてませんでした。

修正しましたのでご確認をお願いします！